ts/DataGrid/DataGrid.ts

```diff
@@ -253,6 +259,8 @@ class DataGrid {
             afterLoadCallback?.(this);
         });
 
+        this.time = new Time({}, this.options?.lang);
```

Comment:
Shouldn't this be equivalent to Chart.ts? In [Chart.ts@init](https://github.com/highcharts/highcharts/blob/e1818767c1546a63f55930855fc72f7c6c275aff/ts/Core/Chart/Chart.ts#L519-L530) we also pass `options.time` and `this.locale`. Without it, the data grid will not respect per-instance language options or locale settings.